java/ql/src/semmle/code/FileSystem.qll

```diff
@@ -146,9 +146,11 @@ class Container extends @container, Top {
   }
 
   /**
-   * Gets a textual representation of the path of this container.
+   * Gets a textual representation of this container.
    *
-   * This is the absolute path of the container.
+   * The default implementation returns the absolute path to the container, but subclasses may
+   * may override to provide a different result. To get the absolute path for any `Container`, call
```

Comment:
The sentence contains "may" twice:
```suggestion
   * override to provide a different result. To get the absolute path for any `Container`, call
```

Maybe there is also something missing after "override"? (might need line wrapping)
```suggestion
   * override this predicate to provide a different result. To get the absolute path for any `Container`, call
```